src/pudl/metadata/fields.py

```diff
@@ -4220,6 +4220,101 @@
         "type": "number",
         "description": "The lower bounds of the turbulence intensity at the wind site (ratio of standard deviation of fluctuating wind velocity to the mean wind speed).",
     },
+    "azimuth_angle": {
+        "type": "number",
+        "description": "Indicates the azimuth angle of the unit for fixed tilt or single-axis technologies.",
```

Comment:
I almost took time to write out a definition of azimuth angle, but it felt hard to explain it succinctly and also it has its own wikipedia page that explains it quite well.

Reply:
That's fine!

Reply:
It might be good to include a unit suffix in this name. Depending on the context angles can be measured in degrees, radians, or grads.